packages/grafana-data/src/transformations/transformers/joinByField.ts

```diff
@@ -36,6 +36,10 @@ export const joinByFieldTransformer: SynchronousDataTransformerInfo<JoinByFieldO
     return (data: DataFrame[]) => {
       if (data.length > 1) {
         if (options.byField && !joinBy) {
+          // Either "time" or "Time" can be saved in the panel JSON from different versions of Grafana, so we need to make sure that they are both treated the same
+          if (options.byField.toLowerCase() === 'time') {
+            options.byField = 'Time';
```

Comment:
So this solution is not ideal because this value needs to match the default value set in the backend.

Reply:
This change is looks Grafana wide for all data sources and dashboards

Reply:
Yeah I'm just playing around, I don't think we need to be making any frontend changes for this PR